backend/src/packages/chaiNNer_standard/image/create_images/create_colorwheel.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+import navi
+from nodes.impl.gradients import (
+    conic_gradient,
+)
+from nodes.properties.inputs import (
+    NumberInput,
+)
+from nodes.properties.outputs import ImageOutput
+
+from .. import create_images_group
+
+
+@create_images_group.register(
+    schema_id="chainner:image:create_colorwheel",
+    name="Create Colorwheel",
+    description="Create an image with a color wheel.",
+    icon="MdLens",
+    inputs=[
+        NumberInput("Size", minimum=1, unit="px", default=512),
+    ],
+    outputs=[
+        ImageOutput(
+            image_type=navi.Image(  # Creates a square output buffer from scratch and sets it to the width and height defined by the (Number)Input0 defined earlier
+                width="Input0",
+                height="Input0",
+            )
```

Comment:
This node always returns RGB images, so let's annotate that.
```suggestion
            ),
            channels=3,
```

Reply:
Items add to PR. Thanks for the review!